plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2013,7 +2013,7 @@ elementgui.block.jump_factor=<html>Jump factor:<br><small>Normal jump factor is
 elementgui.block.has_gravity=Has gravity?
 elementgui.block.can_walk_through=Can walk through the block?
 elementgui.block.harvest_tool=Tool able to destroy it:
-elementgui.block.harvest_level=<html>Tool harvest level to break with drop:<br><small>Only works with certain materials (e.g. ROCK)
+elementgui.block.harvest_level=<html>Tool harvest level to break with drop:
```

Comment:
The <html> can be removed since it no longer has a tooltip

Reply:
The?